translate/translate.go

```diff
@@ -117,5 +137,34 @@ func (tc *TranslationContext) translateLink(sys map[string]interface{}) interfac
 		}
 		// Look up asset - but from where???
 	}
+
 	return "ERR"
 }
+
+func (tc *TranslationContext) translateDateField(value interface{}) interface{} {
+	re, err := regexp.Compile(`([0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2})(\+[0-9]{2}:[0-9]{2})?`) // want to know what is in front of 'at'
```

Comment:
Should be able to use more standard date parse and format code here

Reply:
FYI: The other dude wrote this, things just got moved around. 
However; if you hate it that much, lets make changing it a follow up thing.